scripts/srv_downsampler.sh

```diff
@@ -100,25 +110,28 @@ while read RES UNIT CHUNK MASTER; do
 	if [ ! ${RES} = "01" ]; then	# Use plural unit
 		UNIT_NAME="${UNIT_NAME}s"
 	fi
-	DST_FILE=${DST_PREFIX}_${RES}${UNIT}.grd
-	grdtitle="${TITLE} at ${RES} arc ${UNIT_NAME}"
-	# Note: The ${SRC_ORIG/+/\\+} below is to escape any plus-symbols in the file name with a backslash so grdedit -D will work
-	if [ -f ${DST_FILE} ]; then	# Do nothing
-		echo "${DST_FILE} exist - skipping"
-	elif [ "X${MASTER}" = "Xmaster" ]; then # Just make a copy of the master
-		echo "Convert ${SRC_FILE} to ${DST_FILE}=${DST_FORMAT}"
-		gmt grdconvert ${SRC_FILE} ${DST_FILE}=${DST_FORMAT} --IO_NC4_DEFLATION_LEVEL=9
-		remark="Reformatted from master file ${SRC_ORIG/+/\\+} [${REMARK}]"
-		gmt grdedit ${DST_FILE} -D+t"${grdtitle}"+r"${remark}"+z"${SRC_NAME} (${SRC_UNIT})"
-
-	else	# Must downsample to a lower resolution via spherical Gaussian filtering
-		# Get suitable Gaussian full-width filter rounded to nearest 0.1 km after adding 50 meters for noise
-		echo "Down-filter ${SRC_FILE} to ${DST_FILE}=${DST_FORMAT}"
-		FILTER_WIDTH=`gmt math -Q ${SRC_RADIUS} 2 MUL PI MUL 360 DIV $INC MUL 0.05 ADD 10 MUL RINT 10 DIV =`
-		gmt grdfilter ${SRC_FILE} -Fg${FILTER_WIDTH} -D${FMODE} -I${RES}${UNIT} -r${DST_NODES} -G${DST_FILE}=${DST_FORMAT} --IO_NC4_DEFLATION_LEVEL=9 --IO_NC4_CHUNK_SIZE=${CHUNK} --PROJ_ELLIPSOID=Sphere
-		remark="Obtained by Gaussian ${DST_MODE} filtering (${FILTER_WIDTH} km fullwidth) from ${SRC_FILE/+/\\+} [${REMARK}]"
-		gmt grdedit ${DST_FILE} -D+t"${grdtitle}"+r"${remark}"+z"${SRC_NAME} (${SRC_UNIT})"
-	fi
+	for REG in ${DST_NODES}; do # Probably doing both pixel and gridline registered output, except for master */
+		DST_FILE=${DST_PREFIX}_${RES}${UNIT}${REG}.grd
```

Comment:
"earth_relief_01m_g.grd" seems more readable?
```suggestion
		DST_FILE=${DST_PREFIX}_${RES}${UNIT}_${REG}.grd
```